CefSharp.WinForms/ChromiumWebBrowser.cs

```diff
@@ -273,25 +288,76 @@ private static void OnApplicationExit(object sender, EventArgs e)
         /// <summary>
         /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
         /// </summary>
+        public ChromiumWebBrowser()
+        {
+
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ChromiumWebBrowser"/> class.
+        /// </summary>
         /// <param name="address">The address.</param>
-        /// <exception cref="System.InvalidOperationException">Cef::Initialize() failed</exception>
         public ChromiumWebBrowser(string address)
         {
-            if (!Cef.IsInitialized && !Cef.Initialize())
+            Address = address;
+        }
+
+        private void Initialize()
```

Comment:
> The purely managed classes in the CefSharp assembly?
```
new DefaultFocusHandler(this);
new DefaultResourceHandlerFactory();
new BrowserSettings();
new ManagedCefBrowserAdapter(this, false);
```
Keeping these in the constructor causes it to crash

> What about everyone creating the control in code as they do now? Needs to be the same as it is now.
Since the constructor that is currently in use doesn't get called by the designer I made it effectively the same